reactive/docker.py

```diff
@@ -49,6 +49,10 @@ def install():
     charm_path = os.getenv('CHARM_DIR', '')
     install_script_path = os.path.join(charm_path, 'scripts/install_docker.sh')
     check_call([install_script_path])
+
+    opts = DockerOpts()
+    render('docker.defaults', '/etc/default/docker', {'opts': opts.to_s()})
```

Comment:
Not necessarily.

If you're deploying a subordinate service (which is where the skip_install short circuit bit is being used) you'll not want to go poking at the principal charms configuration files. In that particular instance we'd need to introduce a pattern for charms to communicate with the parent and modify config by passing data over the docker-host/api (i forget which) relation.

But estute observation :)